node/engine/messageservice/p2p-message-service/service.go

```diff
@@ -103,25 +111,34 @@ func NewMessageService(ip string, port int, me types.Address, pk []byte, bootPee
 		Addrs: ms.p2pHost.Addrs(),
 	}
 	addrs, err := peer.AddrInfoToP2pAddrs(&peerInfo)
-	if err != nil {
-		panic(err)
-	}
+	ms.checkError(err)
+
 	ms.MultiAddr = addrs[0].String()
 	ms.logger.Info("libp2p node", "multiaddrs", addrs)
 
 	err = ms.setupDht(bootPeers)
-
-	if err != nil {
-		panic(err)
-	}
+	ms.checkError(err)
 
 	return ms
 }
 
 func (ms *P2PMessageService) setupDht(bootPeers []string) error {
 	ctx := context.Background()
+
+	var bootAddrs []peer.AddrInfo
+	for _, p := range bootPeers {
+		addr, err := multiaddr.NewMultiaddr(p)
+		ms.checkError(err)
+
+		peer, err := peer.AddrInfoFromP2pAddr(addr)
+		ms.checkError(err)
+
+		bootAddrs = append(bootAddrs, *peer)
+	}
+
 	var options []dht.Option
 	options = append(options, dht.BucketSize(20))
+	options = append(options, dht.BootstrapPeers(bootAddrs...))
```

Comment:
From GoDoc: `BootstrapPeers` configures the bootstrapping nodes that we will connect to to seed and refresh our Routing Table if it becomes empty.